oauth2/oauth2-client/src/test/java/org/springframework/security/oauth2/client/jackson2/OAuth2AuthenticationTokenMixinTests.java

```diff
@@ -139,7 +139,7 @@ public void deserializeWhenMixinRegisteredThenDeserializes() throws Exception {
 				.isEqualTo(expectedAuthentication.getAuthorizedClientRegistrationId());
 		DefaultOAuth2User expectedOauth2User = (DefaultOAuth2User) expectedAuthentication.getPrincipal();
 		DefaultOAuth2User oauth2User = (DefaultOAuth2User) authentication.getPrincipal();
-		assertThat(oauth2User.getAuthorities().containsAll(expectedOauth2User.getAuthorities())).isTrue();
+		assertThat(oidcUser.getAuthorities().containsAll(expectedOidcUser.getAuthorities())).isTrue();
```

Comment:
Why is this change needed?

Reply:
Apologies! I had manually reverted an automated change there, but made a mistake in copying the before & after lines. This has since been corrected in 650a2fc. Thanks for noticing! Goes to show I shouldn't make manual changes and fix the automation & run again. (: